src/common/pf_snmp.c

```diff
@@ -308,14 +292,50 @@ void pf_snmp_get_system_location (
    pnet_t * net,
    pf_snmp_system_location_t * location)
 {
-   pf_fspm_get_system_location (net, location);
+   char im_location[PNET_LOCATION_MAX_SIZE];
+   const char * directory = net->fspm_cfg.file_directory;
```

Comment:
Use the new function `pf_cmina_get_file_directory() `
It would be great if you could modify also the old entries in this file.

Reply:
Done. All entries in file updated.